packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Event Hooks with ngrok
+excerpt: How to demonstrate Event Hooks using ngrok to expose a local app to the internet.
+layout: Guides
+---
+
+This guide demonstrates how to develop and test Okta Event Hooks with the ngrok utility.
+
+Event Hooks are outbound calls from Okta that can be used to notify your own software systems of events occurring in your Okta org. See [Event Hooks](/docs/concepts/event-hooks/) for an overview.
+
+The ngrok utility exposes local web servers to the internet, and can enable the testing of Okta Event Hooks using a local application, rather than an internet-based production or test external service.
+
+To use this guide, you need the following:
+
+- An Okta developer org. [Create an org for free](https://developer.okta.com/signup/).
+- A local application. Or use the one provided in the following guide.
+- The ngrok utility. Installation details are provided in the following guide.
+
+With those prerequisites satisfied, use the following steps to get up-and-running:
+
+1. [Install ngrok](/#install-ngrok)
+1. [Create a local sample app](/#create-a-local-application)
+1. [Set up an Event Hook](/#create-an-okta-event-hook)
+1. [Preview and test](/#preview-test-and-review-the-event-hook)
+
+## Install ngrok
+
+If you already have ngrok installed, move on to the next step! If not, follow the installation instructions at [https://ngrok.com/download](https://ngrok.com/download).
+
+Some installation notes:
+
+- You do not need to have an account to install and run ngrok, but creating a free account provides more features as well as basic authentication.
+- You can install ngrok directly in your project folder, as documented from the ngrok download page. Or you can install on your system's path directory to be able to run ngrok from any folder. Alternatively, you can install the executable in your favorite local folder, but you'll need the folder path when referencing the tool.
+
+### Run ngrok
+
+After installing ngrok, ensure that it's running by creating a "tunnel" into a local port (8082 in this example). If you installed directly into your project folder, run the following command in your terminal:
+
+```terminal
+> ./ngrok http 8082
+```
+
+or if you installed in your system path:
+
+```terminal
+> ngrok http 8082
+```
+
+or if you used your favorite folder:
+
+```terminal
+> ~/applications/ngrok http 8082
+```
+
+If you see the following content in your terminal, ngrok is running successfully:
```

Comment:
Maybe we should highlight the ngrok URL in the screenshot so it's clear that it will be needed later on when setting up the Okta Event Hook

Reply:
I agree. Noticed this when I re-ran my project this week. I've added a "NOTE" underneath the screen shot to copy this URL, but I'll look at highlighting it in the image as well.